metadata-ingestion-modules/airflow-plugin/tests/integration/test_plugin.py

```diff
@@ -0,0 +1,389 @@
+        # Configure the datahub plugin and have it write the MCPs to a file.
+        "AIRFLOW__CORE__LAZY_LOAD_PLUGINS": "False" if is_v1 else "True",
+        "AIRFLOW__DATAHUB__CONN_ID": datahub_connection_name,
+        f"AIRFLOW_CONN_{datahub_connection_name.upper()}": Connection(
+            conn_id="datahub_file_default",
+            conn_type="datahub-file",
+            host=str(meta_file),
+        ).get_uri(),
+        # Configure fake credentials for the Snowflake connection.
+        "AIRFLOW_CONN_MY_SNOWFLAKE": Connection(
+            conn_id="my_snowflake",
+            conn_type="snowflake",
+            login="fake_username",
+            password="fake_password",
+            schema="DATAHUB_TEST_SCHEMA",
+            extra={
+                "account": "fake_account",
+                "database": "DATAHUB_TEST_DATABASE",
+                "warehouse": "fake_warehouse",
+                "role": "fake_role",
+                "insecure_mode": "true",
+            },
+        ).get_uri(),
+        "AIRFLOW_CONN_MY_SQLITE": Connection(
+            conn_id="my_sqlite",
+            conn_type="sqlite",
+            host=str(tmp_path / "my_sqlite.db"),
+        ).get_uri(),
+        # Convenience settings.
+        "AIRFLOW__DATAHUB__LOG_LEVEL": "DEBUG",
+        "AIRFLOW__DATAHUB__DEBUG_EMITTER": "True",
+        "SQLALCHEMY_SILENCE_UBER_WARNING": "1",
+    }
+
+    if not HAS_AIRFLOW_STANDALONE_CMD:
+        raise pytest.skip("Airflow standalone command is not available")
+
+    # Start airflow in a background subprocess.
+    airflow_process = subprocess.Popen(
+        ["airflow", "standalone"],
+        env=environment,
+    )
+
+    try:
+        _wait_for_airflow_healthy(airflow_port)
+        print("Airflow is ready!")
+
+        # Sleep for a few seconds to make sure the other Airflow processes are ready.
+        time.sleep(3)
+
+        # Create an extra "airflow" user for easy testing.
+        if IS_LOCAL:
+            print("Creating an extra test user...")
+            subprocess.check_call(
+                [
+                    # fmt: off
+                    "airflow", "users", "create",
+                    "--username", "airflow",
+                    "--password", "airflow",
+                    "--firstname", "admin",
+                    "--lastname", "admin",
+                    "--role", "Admin",
+                    "--email", "airflow@example.com",
+                    # fmt: on
+                ],
+                env=environment,
+            )
+
+        # Sanity check that the plugin got loaded.
+        if not is_v1:
+            print("[debug] Listing loaded plugins")
+            subprocess.check_call(
+                ["airflow", "plugins", "-v"],
+                env=environment,
+            )
+
+        airflow_username = "admin"
+        airflow_password = (airflow_home / "standalone_admin_password.txt").read_text()
```

Comment:
Wasn't this just set to "airflow"?

Reply:
added a comment to explain - there's two users